src/mobile/src/ui/views/onboarding/LanguageSetup.js

```diff
@@ -78,6 +75,7 @@ class LanguageSetup extends Component {
         if (!isAndroid) {
             SplashScreen.hide();
         }
+        this.animation.play();
```

Comment:
Do you need this if `autoPlay` is true? https://github.com/iotaledger/trinity-wallet/pull/1303/files#diff-e5262fd35ec893b55169f0df589a9e48R135